libbeat/cmd/export/template.go

```diff
@@ -0,0 +1,68 @@
+package export
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/template"
+)
+
+func GenTemplateConfigCmd(name, beatVersion string, beatCreator beat.Creator) *cobra.Command {
+	genTemplateConfigCmd := &cobra.Command{
+		Use:   "template",
+		Short: "Export index template to stdout",
+		Run: func(cmd *cobra.Command, args []string) {
+			version, _ := cmd.Flags().GetString("es.version")
+			index, _ := cmd.Flags().GetString("index")
+
+			// Make it compatible with the sem versioning
+			if version == "2x" {
+				version = "2.0.0"
```

Comment:
Took it from here https://github.com/elastic/beats/blob/master/dev-tools/cmd/index_template/index_template.go#L40, I guess it's there for historical reasons?